yaml/package.yaml

```diff
@@ -54,6 +54,7 @@ dependencies:
 when:
   condition: "!impl(ghc >= 8.0)"
   dependencies:
+  - fail >= 4.9 && < 4.10
```

Comment:
Would you also be able to add a minor version bump and changelog entry explaining the change?